dashboard/src/features/shared/components/KeyValueTable/KeyValueTable.jsx

```diff
@@ -45,18 +45,31 @@ class KeyValueTable extends React.Component {
   }
 
   render() {
-
     return(
       <Section
         title={this.props.title}
         actions={this.props.actions} >
         <table className={styles.table}>
           <tbody>
-            {this.props.items.map((item) =>
-              <tr key={`${item.label}`}>
-                <td className={styles.label}>{item.label}</td>
-                <td className={styles.value}>{this.renderValue(item)}</td>
-              </tr>
+            {this.props.items.map((item) => {
+              if (item.label == 'Tags') {
```

Comment:
Would it be difficult to change this so that instead of checking for a specific label value, we expand `item` to optionally include an `editUrl` property. If it is defined, then the link component will be included, and point to the given URL.